src/js/App/Sidenav/ExpandableNav.js

```diff
@@ -19,6 +19,7 @@ const ExpandableNav = ({ subItems, onClick, title, id, active, ignoreCase, activ
             isActive={active && subItem.id === activeApp}
             onClick={(event) => onClick(event, subItem)}
             navigate={subItem?.navigate}
+            isPublicBeta={subItem?.isPublicBeta}
```

Comment:
I wouldn't name it `isPublicBeta`, but rather `isBeta` as there is already this field used in the config.

Reply:
Agreed with this, but we should also go through and remove things that are not actually in beta anymore

Reply:
https://github.com/RedHatInsights/cloud-services-config/pull/488

Reply:
@PanSpagetka The `isBeta` in the API field is strictly for the API. I've added a field for `isBeta` in the frontend too. We should check on that one. It's in the PR above